### ISSUE TITLE
Add support for hosted UBUNTU and MACOS

### ISSUE THREAD

**Issue by michaelsogos** [maikvandergaag/msft-extensions#104]
Hi @maikvandergaag 

This is very useful BUILD TASK.
Do you think is possible to improve this one (or new one dedicated), to read a file in LNX environment?

**Comment by maikvandergaag** [comment]
Will label it as feature request and schedule it for a later moment

**Comment by michaelsogos** [comment]
thanks 😊

**Comment by maikvandergaag** [comment]
For which extensions would you like this option?

**Comment by michaelsogos** [comment]
> For which extensions would you like this option?

File content to variable.

**Comment by stefankip** [comment]
Yeah, bummer that it doesn't work on linux. Now I have to create my own I guess.

**Comment by stefankip** [comment]
This is how I solved it:

```
- script: echo "##vso[task.setvariable variable=VARIABLE_NAME]$(cat ./path-to/file.txt)"
  displayName: 'Get variable from file'
```

**Comment by Stoobs** [comment]
> 
> 
> For which extensions would you like this option?

Any chance of Trigger Azure DevOps Pipeline?

**Comment by sebumd** [comment]
+1, I installed the plugin and it looks like it doesn't work in Linux:

#[error]The current operating system is not capable of running this task. That typically means the task was written for Windows only. For example, written for Windows Desktop PowerShell.

**Comment by MitchDart** [comment]
+1 needed this on our linux builds. Would have been very useful. @stefankip thanks for that, going to give it a try.

**Comment by RezaRahmati** [comment]
doesn't work on linux with this error

```
The current operating system is not capable of running this task. That typically means the task was written for Windows only. For example, written for Windows Desktop PowerShell.
```

**Comment by FelipeAtlas** [comment]
Hello, any predictions to release the implementation to work with linux?

**Comment by maikvandergaag** [comment]
Sorry, not at this moment as I'm swamped, and I have to find time to get it working for Linux. Any help is appreciated

**Comment by tomabg** [comment]
linux support for "Trigger Azure DevOps Pipeline" would be great
